jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -367,12 +374,19 @@ def vue_slider_reset(self, event):
         # updating during slide, so update them now
         self.vue_unpause_tables()
 
+    def _on_spectrum_viewer_limits_changed(self, event=None):
+        sv = self.app.get_viewer('spectrum-viewer')
+        self.spectrum_viewer_min = sv.state.x_min
+        self.spectrum_viewer_max = sv.state.x_max
+
+        # Also update the slider range
+        self._auto_slider_range()
+
     def _auto_slider_range(self, event=None):
         if not self.rs_slider_range_auto:
             return
         # if set to auto, default the range based on the limits of the spectrum plot
-        sv = self.app.get_viewer('spectrum-viewer')
-        x_min, x_max = sv.state.x_min, sv.state.x_max
+        x_min, x_max = self.spectrum_viewer_min, self.spectrum_viewer_max
```

Comment:
Oh duh, I meant to refactor the slider to use the traitlet. Thanks for catching that!

I appreciate the desire to avoid calling `_auto_slider_range` directly. On the other hand, I do like how the logic is self contained here. It makes it clearer to me how `_on_spectrum_viewer_limits_changed` is working, and what methods are depending on that event (I'm thinking if in the future we have multiple methods that rely on this message, it makes it clear that this method is just dispatching to each of the methods 🤷)

Reply:
ok, then maybe adding a comment that states it assumes `spectrum_viewer_min/max` are up-to-date and if that ever changes, we need to re-introduce the call to access them from the state directly?

Reply:
How's this? https://github.com/spacetelescope/jdaviz/pull/1327/commits/9eb4a4c38fdeab5ab50da905bd888e66dcea0236